scss/bitstyles/atoms/link/link.stories.mdx

```diff
@@ -4,14 +4,14 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
 
 # Link
 
-Mimic the appearance of standard anchor elements. This allows other interactive elements (e.g. `<button>`) to visually match your links.
+Mimic the appearance of standard anchor by applying a classname. This allows other interactive elements (e.g. `<button>`) to visually match your links.
 
-This is generally a bad practice — if it looks like a link, it should be a link — but it’s sometimes necessary for technical reasons, especially when building a JS app e.g. an element shouldn’t be a link because no url change occurs (this could be when displaying a modal, or cancelling an action).
+This is a slightly bad practice — if it looks like a link, it should be a link — that’s sometimes necessary for technical reasons, generally when building a JS app. Actions triggered by javascript should not use anchor elements because no navigation occurs, instead they should use `button` elements, but sometimes that button needs to look like a link (examples are the trigger to displaying a modal, or a smaller de-emphasised cancel button next to your primary “OK” action).
```

Comment:
```suggestion
This is a slightly bad practice — if it looks like a link, it should be a link — that’s sometimes necessary for technical reasons, generally when building a JS app. Actions triggered by JavaScript should not use anchor elements because no navigation occurs, instead they should use `button` elements. But sometimes that button needs to look like a link (examples are the trigger to displaying a modal, or a smaller de-emphasised cancel button next to your primary “OK” action).
```